### ISSUE TITLE
Tip jar / donations

### ISSUE THREAD

**Issue by virusdave** [FreeAndFairElections/FreeAndFair#8]
While this is not intended for profit, and completely free for users, having a tip jar / donations link in app and website to defray server and development costs would be great.

**Comment by gubatron** [comment]
ideally linked to multiple options for convenience (patreon, gofundme, venmo, paypal, stripe etc.) to donors.
Github allows for donations as one of the options.

Good to have this written down, but in terms of priorities it should probably be done once there's an app almost ready to be launched or after it's launched

**Comment by virusdave** [comment]
> but in terms of priorities it should probably be done once there's an app almost ready to be launched or after it's launched

Agreed, it's basically an afterthought to help defray costs if we actually end up with something usable in time (and people actually want to use it).  Bottom priority.
